### ISSUE TITLE
Error in events.js:187 in 2.275.0 runner

### ISSUE THREAD

**Issue by mabushey** [summerwind/actions-runner-controller#238]
This has been working for the last two weeks. Now I'm getting this:
```
✗  kubectl -n github-runner get all
NAME                                    READY   STATUS             RESTARTS   AGE
pod/organizational-runner-bg7ks-726r5   1/2     CrashLoopBackOff   10         37m
pod/organizational-runner-bg7ks-dfkqd   1/2     Error              11         36m
pod/organizational-runner-bg7ks-n92sz   1/2     CrashLoopBackOff   10         36m
pod/organizational-runner-bg7ks-tlz97   1/2     CrashLoopBackOff   7          18m
```
```
✗  kubectl -n github-runner logs pod/organizational-runner-bg7ks-726r5 runner
...
Started listener process
Started running service

√ Connected to GitHub

2020-12-16 21:34:50Z: Listening for Jobs
Runner update in progress, do not shutdown runner.
Downloading 2.275.0 runner
Waiting for current job finish running.
Generate and execute update script.
Runner will exit shortly for update, should back online within 10 seconds.
renamed '/runner/_diag/SelfUpdate-20201216-211735.log' -> '/runner/_diag/SelfUpdate-20201216-211735.log.failed'
Runner listener exited with error code 4
Runner listener exit with undefined return code, re-launch runner in 5 seconds.
Starting Runner listener with startup type: service
Started listener process
events.js:187
      throw er; // Unhandled 'error' event
      ^
Error: spawn /runner/bin/Runner.Listener ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:264:19)
    at onErrorNT (internal/child_process.js:456:16)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
Emitted 'error' event on ChildProcess instance at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:270:12)
    at onErrorNT (internal/child_process.js:456:16)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: 'ENOENT',
  code: 'ENOENT',
  syscall: 'spawn /runner/bin/Runner.Listener',
  path: '/runner/bin/Runner.Listener',
  spawnargs: [ 'run', '--startuptype', 'service', '--once' ]
}
```

**Comment by mabushey** [comment]
It looks like there is a hot fix in 2.275.1:
https://github.com/myoung34/docker-github-actions-runner/commit/b1dced691a0df0bc3fbad5d024152354f5df14d0

**Comment by donovanmuller** [comment]
@mabushey we were having the same issue. Updated our custom runner image with `2.275.1` as per https://github.com/summerwind/actions-runner-controller/pull/239 and all good now 👍
